src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfoNullable.cs

```diff
@@ -79,6 +81,55 @@ protected override void OnWrite(ref WriteStackFrame current, Utf8JsonWriter writ
             }
         }
 
+        protected override void OnWriteDictionary(ref WriteStackFrame current, Utf8JsonWriter writer)
```

Comment:
How's the test coverage of the new code?

Reply:
Sequence 91.66%, 85.71%. It's hard and unlikely to reach the "if converter is null" branch, but I believe that is the correct behavior.

Reply:
> It's hard and unlikely to reach the "if converter is null"

We should try to come up with a test-case to reach it. If there is no way to do so, and is actually unreachable - delete it?